pilot/pkg/kube/inject/inject.go

```diff
@@ -483,6 +483,7 @@ func directory(filepath string) string {
 
 func injectionData(sidecarTemplate, version string, deploymentMetadata *metav1.ObjectMeta, spec *corev1.PodSpec, metadata *metav1.ObjectMeta, proxyConfig *meshconfig.ProxyConfig, meshConfig *meshconfig.MeshConfig) (*SidecarInjectionSpec, string, error) { // nolint: lll
 	if err := validateAnnotations(metadata.GetAnnotations()); err != nil {
+		log.Infof("Invalid annotations: %v %v\n", err, metadata.GetAnnotations())
```

Comment:
Warning? Here and below.

Reply:
Some of these errors don't go away without user intervention (i.e. fixing the template). We might need to rate limit warnings to avoid too much noise in the logs.

Reply:
The top problem is to show what is wrong so user can fix it. Extra filtering or rate limits are lower priority.